packages/mask/src/plugins/Referral/hooks/useRequiredChainId.tsx

```diff
@@ -0,0 +1,12 @@
+import { useState } from 'react'
+import type { ChainId } from '@masknet/web3-shared-evm'
+
+import { supportedChainIds } from '../constants'
+
+export function useRequiredChainId(currentChainId: ChainId) {
+    const [requiredChainId, setRequiredChainId] = useState(
+        supportedChainIds.includes(currentChainId) ? currentChainId : supportedChainIds[0],
```

Comment:
Doesn't look right. Why you need a useState here?

Reply:
Thanks, correct, it's useless. Removed it.